config/config.yaml

```diff
@@ -215,14 +215,26 @@ tide:
   # Default tide config for all repos in the Jetstack org except cert-manager
   - orgs:
     - jetstack
-    - cert-manager
     excludedRepos:
     - jetstack/cert-manager
     - jetstack/tarmak
     - jetstack/cert-manager-csi
     labels:
     - lgtm
     - approved
+    missingLabels:
+    - do-not-merge
+    - do-not-merge/blocked-paths
+    - do-not-merge/cherry-pick-not-approved
+    - do-not-merge/hold
+    - do-not-merge/work-in-progress
+    - needs-rebase
+  # Default tide config for all repos in the cert-manager org
+  - orgs:
+    - cert-manager
```

Comment:
Will this work given that it's setting config for `jetstack` org projects?

Reply:
the one bot runs on both orgs, this copies the config over

Reply:
Right, but what is the point of the `orgs` list? If it filters which orgs will be considered then I think this config section would do nothing, as it is saying to look at the `cert-manager` org for the projects `jetstack/cert-manager` etc.

Reply:
This line says to apply the "defaults" to all repos in the cert-manager org. jetstack/cert-manager is set as an exception here